gradle.properties

```diff
@@ -1,3 +1,10 @@
+android.enableJetifier=true
+android.injected.testOnly=false
```

Comment:
What is this for?

Reply:
`android.injected.testOnly=false ` can enable install debug apk instantly